test/validation/api/system/system.c

```diff
@@ -520,6 +520,25 @@ static void system_test_info(void)
 
 		if (info.cpu_isa_hw.arm != ODP_CPU_ARCH_ARM_UNKNOWN)
 			CU_ASSERT(info.cpu_isa_sw.arm <= info.cpu_isa_hw.arm);
+
+		if (info.cpu_isa_hw.arm == ODP_CPU_ARCH_ARMV6)
+			printf("        ODP_CPU_ARCH_ARMV6\n");
+		else if (info.cpu_isa_hw.arm == ODP_CPU_ARCH_ARMV7)
+			printf("        ODP_CPU_ARCH_ARMV7\n");
+		else if (info.cpu_isa_hw.arm == ODP_CPU_ARCH_ARMV8_0)
+			printf("        ODP_CPU_ARCH_ARMV8_0\n");
+		else if (info.cpu_isa_hw.arm == ODP_CPU_ARCH_ARMV8_1)
+			printf("        ODP_CPU_ARCH_ARMV8_1\n");
+		else if (info.cpu_isa_hw.arm == ODP_CPU_ARCH_ARMV8_2)
+			printf("        ODP_CPU_ARCH_ARMV8_2\n");
+		else if (info.cpu_isa_hw.arm == ODP_CPU_ARCH_ARMV8_3)
+			printf("        ODP_CPU_ARCH_ARMV8_3\n");
+		else if (info.cpu_isa_hw.arm == ODP_CPU_ARCH_ARMV8_4)
+			printf("        ODP_CPU_ARCH_ARMV8_4\n");
+		else if (info.cpu_isa_hw.arm == ODP_CPU_ARCH_ARMV8_5)
+			printf("        ODP_CPU_ARCH_ARMV8_5\n");
+		else if (info.cpu_isa_hw.arm == ODP_CPU_ARCH_ARMV8_6)
+			printf("        ODP_CPU_ARCH_ARMV8_6\n");
```

Comment:
Perhaps a final else branch with `CU_FAIL()` could be a useful addition (ODP_CPU_ARCH_ARM_UNKNOWN would also have to be handled). This way undefined values would be detected. Same goes for the SW ISA check above.

Reply:
Ack. Please check v2.